src/encoding_implementations/combination_encoding.jl

```diff
@@ -75,6 +75,9 @@ hidefields(::Type{<:CombinationEncoding}) = [:linear_indices, :cartesian_indices
 
 CombinationEncoding(encodings) = CombinationEncoding(encodings...)
 function CombinationEncoding(encodings::Vararg{Encoding, N}) where N
+    if N == 0
```

Comment:
Actually, I would argue that we should check if `N > 1` as the type also doesn't make sense for only 1 encoding; you could use the encoding directly.